packages/volto/package.json

```diff
@@ -215,7 +215,7 @@
     "process": "^0.11.10",
     "promise-file-reader": "1.0.2",
     "prop-types": "15.7.2",
-    "query-string": "7.1.0",
+    "query-string": "^9.0.0",
```

Comment:
This is potentially breaking, if there are add-ons which depend on the old version without explicitly declaring it (pnpm requires them to declare it, but Volto 18 still supports yarn)

Reply:
@davisagli If someone has a problem, they can declare the version in their package. Would that still be a breaking?

This solves an issue where addon tests might use the wrong version. See: https://github.com/plone/cookieplone-templates/issues/236

Reply:
@nileshgulia1 in fact, I experienced this in the past, but I was surprised that this is gone. Which Node version are you using? Maybe it's that. Please note, that we no longer test 18.

@wesleybl remember that I told you that this package is "special"? It's a Sindre Sorhus package, this means that he's not supporting anything else but ESM packaging, thus, "old" things like Volto has to be specially careful. However, the problem does not show up...

@nileshgulia1 can you confirm?

Reply:
The issue was with older node patch versions. Works in Node v20 and v22.

Reply:
@wesleybl yes I'm using Node 20, but the issue was merely with some older minor version, 20.x. which doesn't support common js require. Now I upgraded to v20.14.0. Everything works well.

Reply:
> yes I'm using Node 20, but the issue was merely with some older minor version, 20.x. which doesn't support common js require

@nileshgulia1 @davisagli @sneridagh can this be considered a breaking change? Volto 18 is not tested with Node 20.0.0, for example.

Reply:
@davisagli So we can't do this on Volto 18?

Reply:
@davisagli @sneridagh can we then make [@plone/client](https://github.com/plone/volto/blob/04aded248a4ce1f8c8ccdf2853ccb4dd1e5733d5/packages/client/package.json#L62) use `query-string` 7 in Volto 18?